### PR TITLE
Improved go.md

### PR DESCRIPTION
**Pull request by beam41**

### CODE REVIEW

File: go.md [hunk 1]
```diff
@@ -30,42 +30,42 @@ func main() {
...
+|-----|-----|-----|----|----|
+|uint8|8-bit unsigned integer|1 byte|0 to 255|byte|
 |int8|8-bit signed integer|1 byte|-128 to 127|
 |int16|16-bit signed integer|2 bytes|-32768 to 32767|
 |unit16|16-bit unsigned integer|2 bytes|0 to 65,535|
-|int32|32-bit signed integer|4 bytes|-2,147,483,648 to 2,147,483,647|
-|uint32|32-bit unsigned integer|4 bytes|0 to 4,294,967,295|
-|int64|64-bit signed integer|8 bytes|	-9,223,372,036,854,775,808 to 9,223,372,036,854,775,807|
-|uint64|64-bit unsigned integer|8 bytes|0 to 18,446,744,073,709,551,615|
+|int32|32-bit signed integer|4 bytes|-2,147,483,648 to 2,147,483,647|int (on 32 bit systems), rune
```
Comment (onecompiler-ops) [inline go.md:56]: end pipe is missing

File: go.md [hunk 2]
```diff
@@ -30,42 +30,42 @@ func main() {
...
 
 ### 1. Numeric Data types
 
 #### Integer Data types
-| Data type | Description | Size|Range|
-|-----|-----|-----|----|
-|uint8|8-bit unsigned integer|1 byte|0 to 255|
+| Data type | Description | Size|Range|Alias|
+|-----|-----|-----|----|----|
+|uint8|8-bit unsigned integer|1 byte|0 to 255|byte|
```
Comment (onecompiler-ops) [inline go.md:52]: What is Alias?

**Review by onecompiler-ops — CHANGES_REQUESTED** [review]

File: go.md (hunk 1 above)
Comment (beam41) [reply go.md:56]: will fix

File: go.md (hunk 2 above)
Comment (beam41) [reply go.md:52]: go type have alternate name for them. you can declare as `uint8` or `byte` it will have same type.

Comment (onecompiler-ops) [reply go.md:52]: Thanks, @beam41 could you please add `NA` for the column if Alias does not exist?

Comment (beam41) [reply go.md:52]: Sure

**Review by onecompiler-ops — APPROVED** [review]

**Comment by onecompiler-ops** [comment]
@beam41 thank you for the contribution

**Pull request merged**